src/woo_publications/publications/tests/test_migrations.py

```diff
@@ -0,0 +1,134 @@
+from datetime import date
+
+from django_test_migrations.contrib.unittest_case import MigratorTestCase
+
+from woo_publications.logging.constants import Events
+
+
+class TestDocumentOwnerMigration(MigratorTestCase):
+    migrate_from = ("publications", "0020_topic_afbeelding")
+    migrate_to = ("publications", "0021_document_eigenaar_publication_eigenaar")
+
+    def prepare(self):
+        """Prepare some data before the migration."""
+        InformationCategory = self.old_state.apps.get_model(
+            "metadata", "informationcategory"
+        )
+        Organisation = self.old_state.apps.get_model("metadata", "organisation")
+        Publication = self.old_state.apps.get_model("publications", "publication")
+        Document = self.old_state.apps.get_model("publications", "document")
+        TimelineLogProxy = self.old_state.apps.get_model("logging", "timelinelogproxy")
+        ContentType = self.old_state.apps.get_model("contenttypes", "contenttype")
+
+        ic = InformationCategory.objects.create(
+            order=1, naam="test", bron_bewaartermijn="Selectielijst gemeenten 2020"
+        )
+        org = Organisation.objects.create(naam="test", is_actief=True)
+        pub_ct, _ = ContentType.objects.get_or_create(
+            app_label="publications", model="publication"
+        )
+        doc_ct, _ = ContentType.objects.get_or_create(
+            app_label="publications", model="document"
+        )
+
+        pub1 = Publication.objects.create(
+            publisher=org,
+            officiele_titel="first pub",
+        )
+        pub1.informatie_categorieen.set([ic])
+        pub1.save()
+
+        TimelineLogProxy.objects.create(
+            content_type=pub_ct,
+            object_id=pub1.pk,
+            extra_data={
+                "event": Events.create,
+                "acting_user": {
+                    "identifier": "identifier-1",
```

Comment:
missed chance for `"905"`

Reply:
905?

Reply:
https://www.reddit.com/r/Counterparts/comments/yckjto/905_meaning/